slack-message.c

```diff
@@ -366,13 +449,52 @@ void slack_json_to_html(GString *html, SlackAccount *sa, json_value *message, Pu
 			slack_attachment_to_html(html, sa, attachments->u.array.values[i]);
 }
 
+void slack_write_message(SlackAccount *sa, SlackObject *obj, const char *html, PurpleMessageFlags flags) {
```

Comment:
I'm not sure why this function exists.  It seems like everywhere it's used should use `purple_conversation_write` instead since it's only for "system" messages, but maybe I'm missing something.

Reply:
To be perfectly honest, I don't remember why I added this, it's almost a year ago. You might be right that it can be replaced with `purple_conversation_write`.

But there is one place where it's used with `PURPLE_MESSAGE_SEND`, IOW not a system message. Is that still ok to use with `purple_conversation_write`?